.github/workflows/sync-dependencies.yml

```
@@ -0,0 +1,42 @@
+          import toml
+          with open("pyproject.toml") as f:
+              data = toml.load(f)
+          deps = data["tool"]["poetry"]["group"]["template"]["dependencies"]
+          print("\n".join([f"{k} = \"{v}\"" if isinstance(v, str) else f"{k} = {v}" for k, v in deps.items()]))
+          ')
+
+          # Update template pyproject.toml
+          cd {{cookiecutter.project_slug}}
+          poetry add $DEPS
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.4)

the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)